vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -104,7 +106,9 @@
     register: provisioned_machines
     retries: 50
     delay: 20
-    until: provisioned_machines.resources | filter_phase("running") | length == (NUMBER_OF_BMH | int)
+    until: (provisioned_machines is succeeded) and
+           (provisioned_machines.resources | length > 0) and
```

Comment:
I don't think we have to check that the length of the resources is > 0. The filter should handle an empty list fine.

Reply:
Right, that's true! Fixing...

Reply:
Done